jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java

```diff
@@ -45,84 +43,88 @@
 
     @Inject
     @DataField("popover")
-    private Popover popover;
```

Comment:
I understand the reason is https://github.com/AppFormer/uberfire/pull/756 provides a more flexible way of using Patternfly popovers on different elements, please correct me if I'm wrong